README.md

```diff
@@ -6,21 +6,27 @@ requirements of LHC-HL upgrade.
 
 # Installation and development
 
-This project requires third-party libraries (TPLs) to build a fully-featured
-code. However, some of these requirements can be omitted to enable testing on
-personal machines with fewer available components.
-
-- ROOT: for I/O
-- an MPI implementation (such as OpenMPI): for shared memory parallelism
-- VecGeom: on-device navigation of GDML or ROOT-defined detector geometry
-- CUDA: on-device computation.
-
-Development/testing requirements:
-
-- CMake (build system)
-- [Git-LFS](https://git-lfs.github.com) (large test files, binary
-  documentation)
-- LLVM/Clang (formatting enforcement)
+This project requires external dependencies to build with full functionality.
+However, any combination of these requirements can be omitted to enable
+limited development on personal machines with fewer available components.
+
+- [CUDA](https://developer.nvidia.com/cuda-toolkit): on-device computation
+- an MPI implementation (such as [Open MPI](https://www.open-mpi.org)): shared-memory parallelism
+- [https://root.cern](ROOT): I/O
+- [https://github.com/nlohmann/json](nljson): simple text-based I/O for
+  diagnostics and program setup
+- [https://gitlab.cern.ch/VecGeom/VecGeom](VecGeom): on-device navigation of GDML-defined detector geometry
+- [https://geant4.web.cern.ch/support/download](Geant4): preprocessing physics data for a problem input
+- [https://geant4.web.cern.ch/support/download](G4EMLOW): EM physics model data
+- [http://hepmc.web.cern.ch/hepmc/](HepMC3): Event input
+- [http://swig.org](SWIG): limited set of Python wrappers for analyzing input
```

Comment:
The markup is reversed here on all the links (should be text in brackets, links in parentheses)

Reply:
🤦 thanks, will fix... interestingly it looks like it still renders correctly on github